packages/client/vite.config.ts

```diff
@@ -90,7 +90,7 @@ export default defineConfig(({ mode }) => {
     },
     resolve: {
       alias: {
-        crypto: 'crypto-js'
+        crypto: require.resolve('crypto-js')
```

Comment:
Do we know why we have this? :D

Reply:
I did try and figure it out but gave up

Reply:
Feel free to remove if E2E's pass imo

Reply:
I don't think the dev environment starts without this